v3/src/v2/import-v2-document.ts

```diff
@@ -55,7 +55,7 @@ export function importV2Document(v2Document: CodapV2Document) {
           const _zIndex = zIndex != null ? { zIndex } : {}
           if (zIndex != null && zIndex > maxZIndex) maxZIndex = zIndex
           const _layout: IFreeTileInRowOptions = {
-            x: left, y: top, ..._width, ..._height, ..._zIndex, isHidden, isMinimized
+            x: x ?? left, y: y ?? top, ..._width, ..._height, ..._zIndex, isHidden, isMinimized
```

Comment:
```suggestion
            x: left ?? x, y: top ?? y, ..._width, ..._height, ..._zIndex, isHidden, isMinimized
```
We only want to use `x` and `y` if `left` and `top` are not present.